### PR TITLE
feat: run pkg scripts with or without run cmd

### PR DESCRIPTION
**Pull request by triyanox**
Adds a `run` command for package scripts also enables running scripts without the `run` command.

### CODE REVIEW

File: src/pkg_manager.ts [hunk 1]
```diff
@@ -26,104 +26,121 @@ export interface PackageManager {
...
 
-export type PkgManagerName = "npm" | "yarn" | "pnpm" | "bun";
+export type PkgManagerName = 'npm' | 'yarn' | 'pnpm' | 'bun';
 
 function getPkgManagerFromEnv(value: string): PkgManagerName | null {
-  if (value.startsWith("pnpm/")) return "pnpm";
-  else if (value.startsWith("yarn/")) return "yarn";
-  else if (value.startsWith("npm/")) return "npm";
-  else if (value.startsWith("bun/")) return "bun";
+  if (value.startsWith('pnpm/')) return 'pnpm';
```
Comment (bartlomieju) [inline src/pkg_manager.ts:140]: Please revert all of these formatting changes. Use `deno fmt` to format the code.

Comment (triyanox) [reply src/pkg_manager.ts]: @bartlomieju thank your assistance the code is formatted using `deno fmt`

**Comment by bartlomieju** [comment]
Thanks for the PR, please sign the CLA so we can merge it :)

**Comment by triyanox** [comment]
Hey @bartlomieju I signed the CLA and also resolved the conflicts with `upstream/main`

**Review by marvinhagemeister — APPROVED** [review]
Sweet, thanks for adding this 👍

**Pull request merged**